src/main/kotlin/pt/up/fe/ni/website/backend/model/PerActivityRole.kt

```diff
@@ -26,4 +27,10 @@ class PerActivityRole(
     @ManyToOne
     @JsonBackReference
     lateinit var role: Role
+
+    override fun toString(): String {
+        val activityTitle = activity.title.filter { it.isLetterOrDigit() }.uppercase(Locale.getDefault())
```

Comment:
How is this different from `uppercase()`?

Reply:
Please update, why is locale.getDefault used?

Reply:
The `Locale.getDefault()` makes the `uppercase()` method respect the user's locale settings, some characters in uppercase can differ in other languages. Maybe this isn't necessary but it doesn't hurt to leave it there. What do you think? @BrunoRosendo

Reply:
That makes sense to me!